community/udc/src/main/java/org/neo4j/ext/udc/impl/DefaultUdcInformationCollector.java

```diff
@@ -274,7 +274,7 @@ private long determineTotalMemory()
         {
             return ((OperatingSystemMXBean) operatingSystemMXBean).getTotalPhysicalMemorySize();
         }
-        catch ( NoClassDefFoundError e )
+        catch ( Throwable e )
```

Comment:
That restricts us to only JVMs that we look for. That would be more brittle, as they can change their implementation details. This feature-discovery like method is more robust against that.